azurerm/encryption_settings.go

```diff
@@ -0,0 +1,179 @@
+package azurerm
+
+import (
+	"github.com/Azure/azure-sdk-for-go/arm/compute"
+	"github.com/Azure/azure-sdk-for-go/arm/disk"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func encryptionSettingsSchema() *schema.Schema {
```

Comment:
I'd suggest it might be worth calling this `diskEncryptionSettingsSchema()`?

Reply:
done